envs/dali-env.yaml

```diff
@@ -5,6 +5,7 @@ imported_envs:
 
 packages:
 {% if build_type == 'cuda' %}
+  - feedstock : cudatoolkit    #[cudatoolkit == "11.2"]
```

Comment:
Should it not come from imported tensorflow-env.yaml?

Reply:
yes, this is not strickly needed here, but it doesn't hurt.